tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_brick_verification.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+
+
+class TestBrickVerification(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.homescreen = Homescreen(self.marionette)
+
+    def test_verify_homescreen(self):
+        self.apps.switch_to_displayed_app()
```

Comment:
This works because `GaiaTestCase` kills the FTU at a beginning of a test. People not used to the mechanism of it, but familiar with a flash of a device could say: "What happens if the FTU is displayed?". I'm not that kind of person, but it took me some time to remember if that was the case or not :grin:

I would add a comment there to remove this ambiguity.